go.mod

```diff
@@ -31,7 +31,10 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
-require cosmossdk.io/errors v1.0.0-beta.7
+require (
+	cosmossdk.io/errors v1.0.0-beta.7
+	github.com/cosmos/interchain-security v1.0.0
```

Comment:
This means that ICS v2 has a direct dependency on ICS v1 in all future iterations?
This also means that when you add v3 our imports would look like this (see below)?
```golang
require (
    github.com/cosmos/interchain-security v1.0.0
    github.com/cosmos/interchain-security/v2 v2.0.0
)
```